apis/Google.Cloud.Firestore/Google.Cloud.Firestore.IntegrationTests/FirestoreMultipleDbTest.cs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022, Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.ClientTesting;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Firestore.IntegrationTests

```

Comment:
I should have noticed this before - as this is a new file, let's use file-scoped namespaces. Just add ; at the end of this line, and VS will do the rest. It'll reduce indentation. Ditto the fixture.